ci/jenkins/test-vm.sh

```diff
@@ -170,45 +214,91 @@ function configure_vm_agent {
+      echo "Copying kubeconfig files to Linux VM: $host_name"
+      scp -q -o StrictHostKeyChecking=no -i "${WORKDIR}/jenkins_id_rsa" ${WORKDIR}/${ANTREA_AGENT_ANTREA_KUBECONFIG} ${USERNAME}@${IP_ADDRESS}:/tmp/antrea-ci/${ANTREA_AGENT_ANTREA_KUBECONFIG}
+      scp -q -o StrictHostKeyChecking=no -i "${WORKDIR}/jenkins_id_rsa" ${WORKDIR}/${ANTREA_AGENT_KUBECONFIG}  ${USERNAME}@${IP_ADDRESS}:/tmp/antrea-ci/${ANTREA_AGENT_KUBECONFIG}
+      echo "Copying install script to Linux VM: $host_name"
+      scp -q -o StrictHostKeyChecking=no -i "${WORKDIR}/jenkins_id_rsa" ${WORKDIR}/install-vm.sh  ${USERNAME}@${IP_ADDRESS}:/tmp/antrea-ci/install-vm.sh
+    done
+}
+
+function copy_antrea_agent_files_on_windows {
+    echo "====== Delivering Antrea files to all the WINDOWS VMs ======"
+    for host_name in "${!WINDOWS_HOSTS_TO_IP[@]}"; do
+      echo "Copying binaries and conf to VM: $host_name"
+      USERNAME=${WINDOWS_HOSTS_TO_USERNAME[${host_name}]}
+      IP_ADDRESS=${WINDOWS_HOSTS_TO_IP[${host_name}]}
+      ssh -o StrictHostKeyChecking=no -i "${WORKDIR}/jenkins_id_rsa" ${USERNAME}@${IP_ADDRESS} "mkdir -p /tmp/antrea-ci"
+      scp -q -o StrictHostKeyChecking=no -i "${WORKDIR}/jenkins_id_rsa" ./bin/antrea-agent.exe ${USERNAME}@${IP_ADDRESS}:/tmp/antrea-ci/antrea-agent.exe
+      scp -q -o StrictHostKeyChecking=no -i "${WORKDIR}/jenkins_id_rsa" ./bin/antctl.exe ${USERNAME}@${IP_ADDRESS}:/tmp/antrea-ci/antctl.exe
+      scp -q -o StrictHostKeyChecking=no -i "${WORKDIR}/jenkins_id_rsa" ${WORKDIR}/antrea-agent.conf  ${USERNAME}@${IP_ADDRESS}:/tmp/antrea-ci/antrea-agent.conf
+      echo "Copying kubeconfig files to Windows VM: $host_name"
+      scp -q -o StrictHostKeyChecking=no -i "${WORKDIR}/jenkins_id_rsa" ${WORKDIR}/${ANTREA_AGENT_ANTREA_KUBECONFIG} ${USERNAME}@${IP_ADDRESS}:/tmp/antrea-ci/${ANTREA_AGENT_ANTREA_KUBECONFIG}
+      scp -q -o StrictHostKeyChecking=no -i "${WORKDIR}/jenkins_id_rsa" ${WORKDIR}/${ANTREA_AGENT_KUBECONFIG} ${USERNAME}@${IP_ADDRESS}:/tmp/antrea-ci/${ANTREA_AGENT_KUBECONFIG}
+      echo "Copying install script to Windows VM: $host_name"
+      scp -q -o StrictHostKeyChecking=no -i "${WORKDIR}/jenkins_id_rsa" ${WORKDIR}/install-vm.ps1  ${USERNAME}@${IP_ADDRESS}:/tmp/antrea-ci/install-vm.ps1
+    done
+}
+
+function install_on_linux {
+   for host_name in "${!LINUX_HOSTS_TO_IP[@]}"; do
+     echo "Installing on Linux VM $host_name"
+     USERNAME=${LINUX_HOSTS_TO_USERNAME[${host_name}]}
+     IP_ADDRESS=${LINUX_HOSTS_TO_IP[${host_name}]}
+     ssh -o StrictHostKeyChecking=no -i "${WORKDIR}/jenkins_id_rsa" -n ${USERNAME}@${IP_ADDRESS} "sudo chmod +x /tmp/antrea-ci/install-vm.sh"
+     ssh -o StrictHostKeyChecking=no -i "${WORKDIR}/jenkins_id_rsa" -n ${USERNAME}@${IP_ADDRESS} "cd /tmp/antrea-ci && sudo ./install-vm.sh --ns vm-ns --config ./antrea-agent.conf --kubeconfig ./antrea-agent.kubeconfig  --antrea-kubeconfig ./antrea-agent.antrea.kubeconfig --containerize"
```

Comment:
install-vm.sh downloads antrea-ubuntu images with release tag, so the agent bits we actually used to run test is different from the patch which triggers this CI test ? I don't think this is this change's purpose?

Reply:
This is only for just testing purposes.

Reply:
The CI as of now support only native install.